corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/form.scss

```diff
@@ -34,3 +34,23 @@ legend {
     border-top: $cc-neutral-hi solid 1px;
     padding-top: 20px;
 }
+
+.form-container .form-actions {
+  margin: 30px (-$form-text-indent) 0 (-$form-text-indent);
+  padding: 0px;
+
+  .submit .btn {
+    font-size: 20px;
+    padding: 0.5em 1.5em;
+    transition: all .5s;
+    margin: 0.5em;
```

Comment:
I used `em` here so everything is relative to the font size, which is smaller in live preview (which is really a tiny window)